src/formio/components/AddressNL.jsx

```diff
@@ -368,6 +382,7 @@ const AddressNLForm = ({initialValues, required, deriveAddress, layout, setFormi
 const FormikAddress = ({required, setFormioValues, deriveAddress, layout}) => {
   const {values, isValid, setFieldValue} = useFormikContext();
   const {baseUrl} = useContext(ConfigContext);
+  const [isAddressDisabled, setAddressDisabled] = useState(true);
```

Comment:
I think that the name `isAddressDisabled` could cause some confusion. Maybe something like this describes the situation more clearly:

```suggestion
const [isAddressAutoFilled, setAddressAutoFilled] = useState(true);
```

Reply:
Yes, you are right!It's much better.